src/node_usb.cc

```diff
@@ -297,7 +297,6 @@ void initConstants(Local<Object> target){
 
 Local<Value> libusbException(int errorno) {
 	const char* err = libusb_error_name(errorno);
-	Local<Value> e  = Nan::Error(err);
-	e->ToObject()->Set(Nan::New<String>("errno").ToLocalChecked(), Nan::New<Integer>(errorno));
+	Local<Value> e  = Nan::ErrnoException(errorno, NULL, err, NULL);
```

Comment:
Does `ErrnoException` assume that the value passed is a standard system error code? These are [`libusb_error`](http://libusb.sourceforge.net/api-1.0/group__libusb__misc.html#gab2323aa0f04bc22038e7e1740b2f29ef) or [`libusb_status`](http://libusb.sourceforge.net/api-1.0/group__libusb__asyncio.html#ga9fcb2aa23d342060ebda1d0cf7478856) values.

Reply:
@kevinmehall very good point, I wasn't sure about this part. I've updated the code to keep closer to the original one, and as much as I can tell, it retains the original behaviour, while works on Node 12. I don't have the demo device that is being tested with `full-test` in this repo, but as much as I poked around, it seems to match up.

Does it look better?